.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 99
```

Comment:
Did you mean to include this?

Reply:
Hmm i may of screwed something up when attempting to update from master via a rebase.